### PR TITLE
Move common constants into a separate package

### PR DESCRIPTION
**Pull request by Tal-or**
Signed-off-by: Talor Itzhak <titzhak@redhat.com>

### CODE REVIEW

File: test/e2e/utils/consts.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package utils
```
Comment (fromanirh) [inline test/e2e/utils/consts.go:1]: I like the idea but `utils` is an anti-pattern. Let's think of a better alternative.

Comment (Tal-or) [reply test/e2e/utils/consts.go:1]: How about ```testconsts```?

Comment (Tal-or) [reply test/e2e/utils/consts.go:1]: @fromanirh ^

Comment (fromanirh) [reply test/e2e/utils/consts.go:1]: on second thought the arguments with prevail is avoid kitchen-sink packages.
`testconsts` is a tiny improvement, but moving all the constants into a separate package is a sidegrade after all.
I think the best course of action is to repurpose this PR to add the 
```
// TestNodeLabel unique label for the node that
	TestNodeLabel = "rte-e2e-test-node"
```
constant, and leave the rest of the constants in the `nodes` package. All the constants belong to the node area after all, and overall keeping them there is the lesser evil and a cleanish solution.

Comment (Tal-or) [reply test/e2e/utils/consts.go]: Done

**Review by fromanirh — APPROVED** [review]

**Pull request merged**